dev-docs/source/MantidUsedIconsTable.rst

```diff
@@ -3,6 +3,7 @@
 .. |copy| image:: images/LocalIcons/content-copy.png
 .. |paste| image:: images/LocalIcons/content-paste.png
 .. |cut| image:: images/LocalIcons/content-cut.png
+.. |outline-settings| image:: images/LocalIcons/settings-outline.png
 .. |settings| image:: images/LocalIcons/settings.png
```

Comment:
This is exactly what we should avoid. We only need one settings icon. This already voids the point you made in the guidelines about consistency.

This should probably be follow up work however.

Reply:
Agreed, I am going to remove settings-outline and request @TTitcombe to revisit the ISIS SANS GUI to ensure the page is more in line with current specs including replacing `mdi.play-circle-outline` with `mdi.play` and `mdi.plus-circle-outline` with `mdi.sigma`, which are used in other places.